draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1490,6 +1513,18 @@ uCDN-A                         dCDN-B                         dCDN-C
          implements the filtered collections.  Otherwise, optional.
 	</t>
 
+	<t hangText="Name: all-triggers">
+	<vspace blankLines="1"/>
+	Description: Array of all Trigger Status objects in the collection.
+	Should be returned only when extended trigger collection view is requested as described in 
+	<xref target="polling-collections-extended"/>.
+	<vspace blankLines="1"/>
+	Value: An array of JSON-encoded Trigger Status objects.
+	<vspace blankLines="1"/>
+	Mandatory: No, only if dCDN supports and advertises the appropriate extended 
+	status for trigger collections.
```

Comment:
I might be a bit confused by the wording.  So, this _is_ mandatory if the dCDN does not support and advertise the appropriate extended status for trigger collections?  Seems like it should be the opposite?

Reply:
Fixed the wording.
+       Mandatory: No. The "all-triggers" SHOULD only be used by dCDN that supports and advertises 
+       the appropriate extended status for trigger collections (see <xref target="extended-status-capability-object"/> for detail